src/components/dialog-full-screen/dialog-full-screen.stories.js

```diff
@@ -25,6 +27,12 @@ const handleClick = (evt) => {
   action('click')(evt);
 };
 
+const storyParameters = {
```

Comment:
I think it can be moved to `addParameters` decorator, you won't need to use it in every story

Reply:
I don't want to use it in every story. I want to use it to stories about DialogFullScreen component.

Reply:
Changed